app/uk/gov/hmrc/incorporatedentityidentificationfrontend/api/controllers/JourneyController.scala

```diff
@@ -57,10 +57,11 @@ class JourneyController @Inject()(controllerComponents: ControllerComponents,
         businessVerificationCheck <- (json \ businessVerificationCheckKey).validateOpt[Boolean]
         regime <- (json \ regimeKey).validate[String]
         accessibilityUrl <- (json \ accessibilityUrlKey).validate[String]
+        optFullNamePageLabel <- (json \ optFullNamePageLabelKey).validateOpt[String]
         labels <- (json \ labelsKey).validateOpt[JourneyLabels]
       } yield JourneyConfig(
         continueUrl,
-        PageConfig(optServiceName, deskProServiceId, signOutUrl, accessibilityUrl, labels),
+        PageConfig(optServiceName, deskProServiceId, signOutUrl, accessibilityUrl, optFullNamePageLabel, labels),
```

Comment:
I'm not certain that the optFullNamePageLabel is really required. For sole trader we implemented both the optional full name page label and the optional welsh service name. However, if you look at the example input for incorporated entity in the JIRA ticket SAR-9636 there is only the optional Welsh service name. The partnership frontend is similar to incorporated entity.